data-structures-stacks/is-empty.js

```diff
@@ -0,0 +1,7 @@
+export default function isEmpty(stack) {
+  const data = stack.peek();
+  if (data === undefined) {
```

Comment:
`if (x) return true; else return false;` is pretty ugly code. It isn't code I expect from an experienced developer... I don't think I've ever seen you do it before!